app/ui/errordialog.ui

```diff
@@ -82,6 +82,16 @@
      </item>
     </layout>
    </item>
+   <item>
+    <widget class="QLabel" name="label">
+     <property name="text">
+      <string>The following report contains vital information to figure out the cause of the error. Make sure to copy all of it, if you intend to report the bug.</string>
```

Comment:
I suggest a more concise message: "This report contains vital information. Copy all of it when submitting a bug." Shorter messages often increase the likelihood of a response.